public/pages/createDetector/containers/CreateDetector.tsx

```diff
@@ -68,6 +73,25 @@ export function CreateDetector(props: CreateADProps) {
   //In case user is refreshing Edit detector page, we'll lose existing detector state
   //This will ensure to fetch the detector based on id from URL
   const { detector, hasError } = useFetchDetectorInfo(detectorId);
+  const [sampleCalloutVisible, setSampleCalloutVisible] = useState<boolean>(
+    false
+  );
+  const visibleIndices = useSelector(
+    (state: AppState) => state.elasticsearch.indices
+  ) as CatIndex[];
+
+  // Getting all initial indices
+  useEffect(() => {
+    const getInitialIndices = async () => {
+      await dispatch(getIndices(''));
+    };
+    getInitialIndices();
+  }, []);
+
+  // Check if the sample data callout should be visible based on detector index
+  useEffect(() => {
+    setSampleCalloutVisible(!containsDetectorsIndex(visibleIndices));
```

Comment:
If anomaly detector index not exists in visible indices, we will show sample detector callout on detector creation page? Why not show this callout when click "Create detector" button on dashboard/detector list page? Can you add some screenshot?